modules/services/unison.nix

```diff
@@ -60,7 +62,7 @@ let
     };
   };
 
-  serialiseArg = key: val: escapeShellArg "-${key}=${escape [ "=" ] val}";
+  serialiseArg = key: val: concatStringsSep " " (forEach (toList val) (x: escapeShellArg "-${key}=${escape [ "=" ] x}"));
```

Comment:
Please wrap that line so that it does not exceed 80 chars.

Reply:
(`./format` will do that for you)